frontstage/templates/surveys/surveys-share/send-instructions.html

```diff
@@ -50,16 +50,18 @@
 {% block main %}
 <h1 class="u-fs-xl">Send instructions</h1>
 <p>We will send an email to <b>{{ email }}</b> with instructions to access the following surveys:</p>
-<h2 class="u-fs-m">{{ business_name }}</h2>
-{% if surveys|length > 0 %}
+{% for key in share_dict %}
```

Comment:
What is the `key` representing here?  A business?  Would be nice to replace this name for something more specific

Reply:
👍